tests/test_laspy.py

```diff
@@ -195,8 +195,8 @@ def test_return_num(self):
 
     def test_overflow_return_num(self):
         """Testing overflow handling"""
-        rn = self.FileObject.return_num + 100000
```

Comment:
Did you move this line because it created an OverflowError ?

If yes I think we should do something else because it means the line `self.FileObject.return_num = rn` is no longer really tested